std_msgs/String RectifiedXMLdir 
---